src/functionalTest/resources/application-functional.yaml

```diff
@@ -10,6 +10,8 @@ test.user.password: ${TEST_USER_PASSWORD:Hmcts123}
 s2s.auth.secret: ${S2S_SECRET:OLMEJQA7UHU6EZKG}
 s2s.auth.url: ${S2S_URL_TO_TEST:http://rpe-service-auth-provider-aat.service.core-compute-aat.internal}
 
+s2s-name: rd_user_profile_api
+
```

Comment:
is this needed?

Reply:
All Functional Test cases were failing with 401 when attempting to get S2S Token as it was picking rd-professional-api name and secret - so including the name and secret key resolved the issue